kclvm/runner/src/tests.rs

```diff
@@ -482,7 +482,6 @@ fn test_from_setting_file_program_arg() {
     }
 }
 
-#[test]
```

Comment:
Why remove here.

Reply:
This is an accident from a previous PR #403 , which will run concurrently, increasing the instability of the test process.
This use case is not skipped, it will be executed in method `test_exec()`
```
fn test_exec() {

    test_exec_file();  // Here
    println!("test_exec_file - PASS");

    test_kclvm_runner_execute();
    println!("test_kclvm_runner_execute - PASS");

    test_kclvm_runner_execute_timeout();
    println!("test_kclvm_runner_execute_timeout - PASS");
    fs::remove_dir_all(Path::new("__main__")).unwrap();

    test_custom_manifests_output();
    println!("test_custom_manifests_output - PASS");

    test_exec_with_err_result();
    println!("test_exec_with_err_result - PASS");
}
```